<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Duplicados</class>
 <widget class="QMainWindow" name="Duplicados">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="newItem">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Element:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteDuplicates">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> DD&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="Array1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>201</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="Array2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>201</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKenyStev">
    <property name="title">
     <string>KenyStev</string>
    </property>
   </widget>
   <addaction name="menuKenyStev"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
